distribution/docs/src/main/resources/content/_configuring/connecting-to-external-idp.adoc

```diff
@@ -7,13 +7,19 @@
 
 == {title}
 
-.Service Provider Metadata
-See <<{managing-prefix}configuring_idp_sp, Configuring Security IdP Service Provider>> for details on configuring the IdP client.
+To connect to an external Identity Provider,
+
+. Provide the external IdP with ${branding}'s Service Provider (SP) metadata.
+The SP metadata can found at `https://<FQDN>:<PORT>/services/saml/sso/metadata`.
 
-.Disable Included IdP
-Remove the included IdP.
+. Replace the IdP metadata field in ${branding}.
+.. Navigate to the *${admin-console}*.
+.. Select the *${ddf-security}* application.
+.. Select the *Configuration* tab.
+.. Select *IdP Client*.
+.. Populate the *IdP Metadata* field with the external IdP's metadata.
 
-. Navigate to the *${admin-console}*.
-. Select the *System* tab.
-. Select the *Features* tab.
-. Uninstall `security-idp` feature.
+.Service Provider Metadata
+It is not recommended to remove or replace the included Service Provider.
+To add an external Service Provider, add the SP metadata to the *IdP Server* configuration.
```

Comment:
✏️ To be extra clear with the first line, "To add an **additional,** external Service Provider"